pyproject.toml

```diff
@@ -66,6 +66,7 @@ documentation = "https://numpy.org/doc/"
 source = "https://github.com/numpy/numpy"
 download = "https://pypi.org/project/numpy/#files"
 tracker = "https://github.com/numpy/numpy/issues"
+"Release Notes" = "https://numpy.org/doc/stable/release"
```

Comment:
This does seem to be supported by PyPI and yield an icon: https://github.com/pypi/warehouse/blob/d12deea005f6652862ad0b9d28ca8c3d39bc7402/warehouse/templates/packaging/detail.html#L24

Reply:
Can you make this all lower case though? This fits better with the other content in this file.